### PR TITLE
Ensure version and metamodel version are part of all relevant serializations

### PR DESCRIPTION
**Pull request by deepakunni3**
This PR addresses https://github.com/linkml/linkml/issues/285.

### CODE REVIEW

**Comment by deepakunni3** [comment]
@cmungall @hsolbrig Would also like to hear your thoughts if this is the right approach.

**Comment by deepakunni3** [comment]
So the tests are failing because of "output mismatch". 

If the approach above makes sense then I can fix the tests by updating the outputs that its being compared against.

File: linkml/generators/sqlddlgen.py [hunk 1]
```diff
@@ -204,6 +205,11 @@ def _class_name_to_table(self, cn: ClassDefinitionName) -> str:
         """
         return underscore(cn)
 
+    def generate_header(self):
```
Comment (cmungall) [inline linkml/generators/sqlddlgen.py:208]: are you sure '#' as a comment delimiter is respected by all SQL engines?

Currently in this module we only write DDL via sqlalchemy -- this should take care of emitting comments with the correct syntax

**Review by cmungall — CHANGES_REQUESTED** [review]

File: linkml/generators/sqlddlgen.py (hunk 1 above)
Comment (deepakunni3) [reply linkml/generators/sqlddlgen.py:208]: Thank you for pointing that out @cmungall ! 

I made the change such that we are using the more traditional SQL comment syntax (`/* COMMENT */`)

**Comment by deepakunni3** [comment]
This PR also brings to attention on `generator_version`? I think there is value to keeping track of the version of the generator that was used to autogenerate various artifacts.

But is the generator version updated on changes to the individual generators? Is that part of the LinkML development SOP?

**Review by cmungall — APPROVED** [review]

**Pull request closed without merging**